File: metrics/test/data/test_data_recursion/data_recursion.cs
public class TestClass
{
    public TestClass() {}

    public void recurse(int i)
    {
        recurse(i + 1);
    }

    private void print(string s)
    {
        Console.WriteLine(s);
    }

    public void HelloWorld()
    {
        print("Hello world");
    }
}

File: metrics/test/data/test_data_recursion/data_recursion.java
public class data_recursion {
    public Main() {
    }

    public void recurse(Integer i) {
        recurse(i + 1);
    }

    public void Print(String s) {
        System.out.println(s);
    }

    public void HelloWorld() {
        Print("Hello World");
    }
}
